tests/test_tiff.py

```diff
@@ -49,62 +49,52 @@ def getImage(path, image):
 	assert response.status_code == 200
-	img_response = Image.open(io.BytesIO(response.content))
-	width_resp, height_resp = img_response.size
-	assert width_resp == 256
-	assert height_resp == 256
+	assert response.json()['items'][7]["key"] == 'XResolution'
+	assert response.json()['items'][7]["value"] == '(4294967295, 69255)'
+	assert response.json()['items'][8]['key'] == 'YResolution'
+	assert response.json()['items'][8]["value"] == '(4294967295, 69255)'
+	assert response.json()['items'][10]['key'] == 'ResolutionUnit'
+	assert response.json()['items'][10]["value"] == "CENTIMETER"
 	
-def test_get_norm_tile(app, client):
-	response = client.get("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/tile/zoom/3/ti/15", headers={"accept": "image/jpeg"})
+def test_tiff_norm_tile(client, image_path_tiff):
+	response = client.get(f"/image/upload_test_tiff/{image_path_tiff[1]}/normalized-tile/zoom/3/ti/15", headers={"accept": "image/jpeg"})
 	assert response.status_code == 200
 	
 	img_response = Image.open(io.BytesIO(response.content))
 	width_resp, height_resp = img_response.size
 	assert width_resp == 256
 	assert height_resp == 256
 	
-def test_get_thumb(app, client):
-	response = client.get("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/thumb", headers={"accept": "image/jpeg"})
-	assert response.status_code == 200
-	
-def test_get_resized(app, client):
-	response = client.get("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/resized", headers={"accept": "image/jpeg"})
-	assert response.status_code == 200
+def test_tiff_thumb(client, image_path_tiff):
+	thumb_test(client, image_path_tiff[1], "tiff")
 	
-def test_get_mask(app, client):
-	response = client.post("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/annotation/mask", headers={"accept": "image/jpeg"}, json={"annotations":[{"geometry": "POINT(10 10)"}], "height":50, "width":50})
-	assert response.status_code == 200
+def test_tiff_resized(client, image_path_tiff):
+	resized_test(client, image_path_tiff[1], "tiff")
 	
-def test_get_crop(app, client):
-	response = client.post("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/annotation/crop", headers={"accept": "image/jpeg"}, json={"annotations":[{"geometry": "POINT(10 10)"}], "height":50, "width":50})
-	assert response.status_code == 200
+def test_tiff_mask(client, image_path_tiff):
+	mask_test(client, image_path_tiff[1], "tiff")
 	
-"""	
-def test_crop_null_annot(app, client):
-	response = client.post("/image/upload_test_tiff/earthworm-transv-posterior-to-clitellum-02.tiff/annotation/crop", headers={"accept": "image/jpeg"}, json={"annotations": [], "height":50, "width":50})
-	print(response.__dict__)
-	assert response.status_code == 400
-"""	
+def test_tiff_crop(client, image_path_tiff):
+	crop_test(client, image_path_tiff[1], "tiff")
+
+@pytest.mark.skip
```

Comment:
Could you add the reason why the test is skipped (there is a `reason` keyword argument for `skip`) so that we can remember in the future why it has been disabled :)